docker/Dockerfile.local

```
@@ -0,0 +1,30 @@
+#
+# Usage:
+#   docker/tests/data-sync/build_image.sh
+#
+# The build script compiles the binary in a container with bind-mounted
+# cargo cache, then builds this image to package it.
+
+FROM debian:bookworm-slim
+
+WORKDIR /app
+
+RUN apt-get update && apt-get install -y --no-install-recommends \
```

Hadolint (2.14.0)

[warning] Pin versions in apt get install. Instead of `apt-get install <package>` use `apt-get install <package>=<version>`

(DL3008)